README.md

```diff
@@ -59,14 +59,23 @@ Some highlights:
         </pre>
     * Install Intel MKL - See
        [instructions](https://software.intel.com/content/www/us/en/develop/articles/installing-intel-free-libs-and-python-apt-repo.html).
-       * MKL can also be installed from Intel Parallel Studio XE. Agora has been tested with 2019 and 2020 versions. 
+       * MKL can also be installed from Intel Parallel Studio XE. Agora has been tested with 2019, 2020, and 2021 versions. 
        * **NOTE**: To enable JIT acceleration applied for matrix multiplication in the code, MKL version after 2019 update 3 is required.
 
     * Optional: DPDK
-       * [DPDK](http://core.dpdk.org/download/) version 20.02.1 is tested with
+       * [DPDK](http://core.dpdk.org/download/) version 20.02.1 and 20.11.1 has been tested with
         Intel 40 GbE and Mellanox 100 GbE NICs in Agora.
-       * To install it, run `sudo make install T=x86_64-native-linuxapp-gcc
-        DESTDIR=/usr -j`
+       * It is required you enable hugepage support and run agora under sudo permissions (LD_LIBRARY_PATH=${LD_LIBRARY_PATH}).
+       * Mellanox dpdk support depends on libibverbs / libmlx5 / ibverbs-utils 
+       * To install version 20.02.1 of dpdk, run `sudo make install T=x86_64-native-linuxapp-gcc
+        DESTDIR=/usr -j` (CONFIG_RTE_LIBRTE_MLX5_PMD=y in config/common_base to enabled MLX5 poll mode driver)
+       * To install version 20.11.1 
+        <pre>
+        $ meson build && cd build && ninja
+        $ sudo ninja install
+        $ sudo ldconfig
+        </pre>
+        the MLX poll mode driver will be autodetected and installed if required
```

Comment:
I think we should add a separate README file for DPDK to explain how to install it. This could make the main README file clearer and we can also add more description about DPDK installation in the separate file.

Reply:
Attempted to do this.